src/OrganisationUnitTree/hasDescendantSelectedPaths.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Checks wether there are descendants of a path in the
+ * array of selected paths
+ *
+ * @param {string} path
+ * @param {string[]} selected
+ */
+export const hasDescendantSelectedPaths = (path, selected) =>
+    selected.reduce((hasSelectedDescendantPaths, curPath) => {
+        // already found out that there is at least one descendant
+        if (hasSelectedDescendantPaths) return hasSelectedDescendantPaths
```

Comment:
I don't think that `.reduce` is the correct way to check this. I believe the correct array method would be `.some` in this case. That'll just return once it finds a thruthy case, so you won't have to do the above.